dgl_sparse/include/sparse/sparse_matrix.h

```diff
@@ -182,6 +182,37 @@ class SparseMatrix : public torch::CustomClassHolder {
   c10::intrusive_ptr<SparseMatrix> RangeSelect(
       int64_t dim, int64_t start, int64_t end);
 
+  /**
+   * @brief Create a SparseMatrix by sampling elements based on the specified
+   * dimension and sample count.
+   *
+   * If `ids` is provided, this function samples elements from the specified
+   * set of row or column IDs, resulting in a sparse matrix containing only
+   * the sampled rows or columns.
+   *
+   * @param dim Select rows (dim=0) or columns (dim=1) for sampling.
+   * @param fanout The number of elements to randomly sample from each row or
+   * column.
+   * @param ids An optional tensor containing row or column IDs from which to
+   * sample elements.
+   * @param replace Indicates whether repeated sampling of the same element
+   * is allowed. When 'replace = true', repeated sampling is permitted; when
+   * 'replace = false', it is not allowed.
+   * @param bias An optional boolean flag indicating whether to enable biasing
+   * during sampling. When 'bias = true', the values of the sparse matrix will
+   * be used as bias weights.
```

Comment:
If True, the values of the sparse matrix will be used as bias weights, meaning that elements with higher values will be more likely to be sampled. Otherwise, all elements will be sampled uniformly, regardless of their value.